x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -38,6 +38,10 @@ function useServicesFetcher() {
     query: { rangeFrom, rangeTo, environment, kuery },
   } = useAnyOfApmParams('/services/{serviceName}', '/services');
 
+  const {
+    query: { serviceGroup },
+  } = useAnyOfApmParams('/services');
```

Comment:
`useAnyOfApmParams` is intended for matching multiple route paths. It's separate from `useApmParams` because it carries a bigger perf penalty. I think `useApmParams` is more appropriate here.